include/modem/lte_lc.h

```diff
@@ -1189,6 +1199,23 @@ int lte_lc_periodic_search_clear(void);
  */
 int lte_lc_periodic_search_request(void);
 
+/** @brief Read current reduced mobility mode.
+ *
+ * @param[out] mode pointer to where the current reduced mobility mode should be written to
+ * @retval 0 if a mode was found and written to the provided pointer.
+ * @retval -EINVAL if input parameter was NULL.
+ * @retval -EFAULT if an AT command failed.
+ */
+int lte_lc_reduced_mobility_get(enum lte_lc_reduced_mobility_mode *mode);
+
+/** @brief Set reduced mobility mode.
+ *
```

Comment:
Could add a note here stating from which modem fw version this AT command is available.